quicklogic_timings_importer/quicklogic_timings_importer.py

```diff
@@ -164,8 +171,13 @@ def parsesetuphold(cls, delval_rise, delval_fall, objectname, entrydata):
             'removal_rising': ('removal', 'posedge'),
             'recovery_falling': ('recovery', 'negedge'),
             'recovery_rising': ('recovery', 'posedge'),
-            'clear': None,
         }
+
+        if cls.normalize_port_names:
+            normalize = cls.normalize_name
+        else:
+            normalize = lambda x: x
```

Comment:
you're doing this check here and in `parseiopath` can you move this selection to a separate function so we do not duplicate the code?